### PR TITLE
[MRG] ICA indexing

### PR DESCRIPTION
**Pull request by jaeilepp**
Takeover from https://github.com/mne-tools/mne-python/pull/3328

### CODE REVIEW

File: mne/viz/ica.py [hunk 1]
```diff
@@ -130,74 +106,6 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
     return fig
 
 
-def _plot_ica_grid(sources, start, stop,
```
Comment (jaeilepp) [inline mne/viz/ica.py]: This function was not used anywhere (along with the callback above) AFAIK.

File: mne/viz/topomap.py [hunk 2]
```diff
@@ -825,7 +825,7 @@ def plot_ica_components(ica, picks=None, ch_type=None, res=64,
     if merge_grads:
         from ..channels.layout import _merge_grad_data
     for ii, data_, ax in zip(picks, data, axes):
-        ax.set_title('IC #%03d' % ii, fontsize=12)
+        ax.set_title('IC #%03d' % (ii + 1), fontsize=12)
```
Comment (agramfort) [inline mne/viz/topomap.py]: in this case if you write # than it points to the index of the component so no +1

makes sense?

**Comment by jaeilepp** [comment]
I think that's all. Ready for review.

**Comment by agramfort** [comment]
thx @jaeilepp !

**Pull request merged**